### ISSUE TITLE
Rückmeldung von SPAM-Anrufen

### ISSUE THREAD

**Issue by haumacher** [haumacher/phoneblock#5]
Aktuell hat PhoneBlock keine Möglichkeit, zu Erkennen, ob von einer Rufnummer immer noch SPAM-Anrufe ausgehen. Die Fritz!Boxen registrieren die abgelehnten Anrufe zwar in der Anrufliste, haben aber keine Möglichkeit, PhoneBlock darüber zu informieren. Mit einer App auf dem Mobiltelefon könnten Nutzer die blockierten Anrufe von ihrer Fritz!Box abrufen und an PhoneBlock zurückmelden. Nur so kann die Blocklist vernünftig in der Länge beschränkt werden. 

Für den Abruf muss das Mobiltelefon in das Heimnetz/WLAN eingebucht sein. Vgl. auch #4 .

**Comment by pfichtner** [comment]
Mit der Infrastuktur die Du nun mit phoneblock-ab hast wäre dies möglich: 
Wenn sich auf dem gleichen Weg phoneblock-ab mit der jeweiligen FritzBox verbindet wäre PhoneBlock zu erkennen, welche Nummern noch aktiv sind und welche nicht. 
Das ganze ist/wäre dann auch nicht auf Fritz!Boxen beschränkt.

**Comment by pfichtner** [comment]
Von denjenigen, die den AB nutzen bekommst Du ja bereits die Rückmeldung. 
Für Nutzende, die den AB zwar nicht nutzen wollen, aber bereit sind die angerufenen Nummern mitzuteilen wäre ein Quick-fix: Mindestkonfidenz "999 - Nie annehmen" im AB als Auswahl anzubieten.

**Comment by haumacher** [comment]
Ja, der Anrufbeantworter liefert eine Rückmeldung - insofern ist das Ticket eigentlich erledigt. Leider ist der AB aber auch um ein Vielfaches komplizierter zu installieren und kann Bedenken bezüglich der Privatsphäre hervorrufen, weil damit natürlich jeder Anruf zumindest eine Datenspur zum AB hinterlässt - nicht nur von SPAM-Anrufen.

Bei den Installationen führt die Blockliste auf jedenfall die Zahlen haushoch an - leider.
